scenarios/ecs_takeover/terraform/ecs.tf

```diff
@@ -1,3 +1,130 @@
 resource "aws_ecs_cluster" "ecs_cluster" {
-    name  = "my-cluster"
+  name = "${var.scenario-name}-${var.cgid}-cluster"
+}
+
+resource "aws_ecs_task_definition" "vault" {
+  family = "${var.scenario-name}-${var.cgid}-vault"
+
+  # Wait for the website to be deployed to the cluster.
+  # This should make sure the instances are available.
+  container_definitions = jsonencode([
+    {
+      name      = "vault"
+      image     = "busybox:latest"
+      essential = true
+      cpu       = 50
+      memory    = 50
+      command   = ["/bin/sh -c \"echo '{{FLAG_1234677}}' >  /FLAG.TXT; sleep 365d\""]
+      entryPoint = [
+        "sh",
+        "-c"
+      ]
+    }
+  ])
+}
+
+// Hosts role we want to use to force reshced
+resource "aws_ecs_task_definition" "privd" {
+  family        = "cg-${var.scenario-name}-${var.cgid}-vault"
+  task_role_arn = aws_iam_role.privd.arn
+  container_definitions = jsonencode([
+    {
+      name      = "privd"
+      image     = "busybox:latest"
+      cpu       = 50
+      memory    = 50
+      essential = true
+      command   = ["sleep", "365d"]
+    }
+  ])
+}
+
+// Hosts website to container escape
+resource "aws_ecs_task_definition" "vulnsite" {
+  family       = "cg-${var.scenario-name}-${var.cgid}-vulnsite"
+  network_mode = "host"
+  container_definitions = jsonencode([
+    {
+      name         = "vulnsite"
+      image        = "docker.io/seb1055/vulnsite:latest"
```

Comment:
I think we'll need to update this to our docker hub before this is merged. That's not pushed yet so left as is for now.

Reply:
Agreed the image should be pulled from the Rhino hub